Typescript: Compile error when using getBufferAsync(Jimp.AUTO)

<!-- IF YOU DO NOT FILL OUT THE FOLLOWING FORM YOUR ISSUE WILL AUTOMATICALLY BE CLOSED!!! -->

# Expected Behavior
`getBufferAsync(Jimp.AUTO)` should be allowed as per package README

> If Jimp.AUTO is passed as the MIME type then the original MIME type for the image (or "image/png") will be used. Alternatively, image.getMIME() will return the original MIME type of the image (or "image/png").

<!-- Please describe the behavior you are expecting -->

# Current Behavior
VSCode will report problem and tsc will throw compile error


<!-- What is the current behavior? -->

# Failure Information (for bugs)
Compiler output:
> error TS2345: Argument of type '-1' is not assignable to parameter of type 'string'.

<!-- Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template. -->

## Steps to Reproduce
npm install -g typescript
npm install --save jimp
npm install @types/node --save-dev
tsc -p tsconfig.json

<!-- Please provide detailed steps for reproducing the issue. Or provide a small runnable code example.

IF YOUR ISSUE DEPENDS ON A PARTICULAR IMAGE BE SURE TO INCLUDE THIS AS WELL. WE CAN'T REPORDUCE IF WE DON'T HAVE YOUR IMAGE

**Screenshots**
<!-- If applicable, add screenshots to help explain your problem. -->

## Context
In the `jimp.d.ts`, these functions should be updated to allow number type for `mime` parameter.
```
 getBase64(mime: string, cb: GenericCallback<string, any, this>): this;
 getBase64Async(mime: string): Promise<string>;
 getBuffer(mime: string, cb: GenericCallback<Buffer>): this;
 getBufferAsync(mime: string): Promise<Buffer>;
```

<!-- Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions. -->

- Jimp Version: 0.10.2
- Operating System: macOS 10.14.6
- Node version: 12.16.1

## Failure Logs

--- a/packages/core/types/jimp.d.ts
+++ b/packages/core/types/jimp.d.ts
@@ -139,8 +139,8 @@ export interface Jimp {
   write(path: string, cb?: ImageCallback<this>): this;
   writeAsync(path: string): Promise<this>;
   rgba(bool: boolean, cb?: ImageCallback<this>): this;
-  getBase64(mime: string, cb: GenericCallback<string, any, this>): this;
-  getBase64Async(mime: string): Promise<string>;
+  getBase64(mime: string | JimpConstructors["AUTO"], cb: GenericCallback<string, any, this>): this;
+  getBase64Async(mime: string | JimpConstructors["AUTO"]): Promise<string>;
   hash(cb?: GenericCallback<string, any, this>): string;
   hash(
     base: number | null | undefined,

